src/hotspot/cpu/x86/compressedKlass_x86.cpp

```diff
@@ -27,20 +27,31 @@
 
 #include "oops/compressedKlass.hpp"
 #include "utilities/globalDefinitions.hpp"
+#include "memory/metaspace.hpp"
```

Comment:
`make test TEST=sources/` would likely complain about unsorted order.

Reply:
Ah no, it only checks this directories:

```
    private static final String[] HOTSPOT_SOURCES_TO_CHECK = {
                    "share/c1",
                    "share/ci",
                    "share/compiler",
                    "share/jvmci",
    };
```

Still worthwhile to sort the includes right here.

Reply:
Fixed